docs/accounts/dev-process.md

```diff
@@ -365,3 +368,98 @@ is to:
    remember to fix up the changelog before pushing
    if required.
 
+## Security issues
+
+Since most of our work happens in the open,
+we need special procedures
+for dealing with security-sensitive issues
+that must be fixed in production
+before being made visible to the public.
+
+We use private bugzilla bugs
+for tracking security-related issues,
+because this allows us to manage visibility
+for other stakeholders at Mozilla
+while maintaining confidentiality.
+
+We use private github repos
+for developing security fixes
+and tagging security-related releases.
+
+### Filing security issues
+
+If you believe you have found
+a security-sensitive issue
+with any part of the Firefox Accounts service,
+please file it as confidential security bug
+in Bugzilla via this link:
+
+* [File a security-sensitive FxA bug](https://bugzilla.mozilla.org/enter_bug.cgi?product=Cloud%20Services&component=Server:%20Firefox%20Accounts&groups=cloud-services-security)
+
+The Firefox Accounts service
+is part of Mozilla's [bug bounty program](https://www.mozilla.org/en-US/security/bug-bounty/),
```

Comment:
Can you remove the `en-US` from these two links so contributors get the correct page for their locale?